src/jit/assertionprop.cpp

```diff
@@ -2852,16 +2852,15 @@ GenTreePtr Compiler::optAssertionProp_LclVar(ASSERT_VALARG_TP assertions, const
     }
 
     BitVecOps::Iter iter(apTraits, assertions);
-    unsigned        index = 0;
-    while (iter.NextElem(&index))
+    for (int i = iter.NextElem(); i != -1; i = iter.NextElem())
     {
```

Comment:
true, but I'm not worried about that. Especially if a more descriptive name than `i` is used.